internal/controllers/nmc_reconciler.go

```diff
@@ -993,8 +1014,32 @@ func (p *podManagerImpl) baseWorkerPod(
 			MountPath: "/usr/lib/modules",
 			ReadOnly:  true,
 		},
+		{
+			Name:      volNameTmp,
+			MountPath: sharedFilesDir,
+			ReadOnly:  true,
+		},
 	}
 
+	const initContainerArgTmpl = `
+mkdir -p {{.KmodsHostPath}};
+cp -R {{.KmodsPath}} {{.KmodsHostPath}};
```

Comment:
KmodsPath -> containerImageDirName?

Reply:
`containerDirName` was a good name when we were mounting the whole `containerImage` into a `DirName` which is not the case anymore. Here we are referring to the kmods path.

Reply:
How about containerImagePath and WorkerImagePath? This way it becomes clear what we are talking about: copying from containerImage into the Worker Container

Reply:
Fixed.